hack/istio/install-music-store-demo.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/bash
+
+##############################################################################
+# install-music-store-demo.sh
+#
+# Installs the kiali music store demo application
+# https://github.com/kiali/demos/tree/master/music-store
+# Works on both openshift and non-openshift environments.
+##############################################################################
+
+: ${CLIENT_EXE:=oc}
+: ${DELETE_DEMOS:=false}
+: ${MSTORE:=music-store}
+: ${BASE_URL:=https://raw.githubusercontent.com/kiali/demos/master}
+
+apply_network_attachment() {
+  NAME=$1
+cat <<NAD | $CLIENT_EXE -n ${NAME} apply -f -
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: istio-cni
+NAD
+    cat <<SCC | $CLIENT_EXE apply -f -
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
+  name: ${NAME}-scc
+runAsUser:
+  type: RunAsAny
+seLinuxContext:
+  type: RunAsAny
+supplementalGroups:
+  type: RunAsAny
+users:
+- "system:serviceaccount:${NAME}:default"
+- "system:serviceaccount:${NAME}:${NAME}"
+SCC
+}
+
+install_mstore_app() {
+
+  if [ "${IS_OPENSHIFT}" == "true" ]; then
+    ${CLIENT_EXE} get project ${MSTORE} || ${CLIENT_EXE} new-project ${MSTORE}
+  fi
+
+  ${CLIENT_EXE} get ns ${MSTORE} || ${CLIENT_EXE} create ns ${MSTORE}
+  ${CLIENT_EXE} label namespace ${MSTORE} istio-injection=enabled --overwrite=true
+  ${CLIENT_EXE} apply -f https://raw.githubusercontent.com/leandroberetta/demos/master/music-store/ui.yaml -n ${MSTORE}
+  ${CLIENT_EXE} apply -f https://raw.githubusercontent.com/leandroberetta/demos/master/music-store/backend.yaml -n ${MSTORE}
+
+  if [ "${IS_OPENSHIFT}" == "true" ]; then
+    apply_network_attachment ${MSTORE}
+  fi
+  ${CLIENT_EXE} wait --timeout 60s --for condition=available deployment/music-store-backend-v1 -n music-store
+  ${CLIENT_EXE} wait --timeout 60s --for condition=available deployment/music-store-ui-v1 -n music-store
+
+  export INGRESS_PORT=$(${CLIENT_EXE} -n istio-system get service istio-ingressgateway -o jsonpath='{.spec.ports[?(@.name=="http2")].nodePort}')
+  if [ "${IS_OPENSHIFT}" == "true" ]; then
+    export INGRESS_HOST=$(crc ip)
+  else
+    export INGRESS_HOST=$(minikube ip)
```

Comment:
This line breaks for me because my minikube profile is not the default `minikube`:

```
You should be able to access the API with the url: http://🤷  Profile "minikube" not found. Run "minikube profile list" to view all profiles.
👉  To start a cluster, run: "minikube start":30755/api in order to generate traffic
You should be able to access the UI with the url: http://🤷  Profile "minikube" not found. Run "minikube profile list" to view all profiles.
👉  To start a cluster, run: "minikube start":30755/ in order to generate traffic
```
We should allow the user to pass in an option `--minikube-profile` (`--mp`) and use that (its default should be `minikube`). We do this in other hack scripts so you can just copy what we do there. For example, in the bookinfo demo install hack script see: https://github.com/kiali/kiali/blob/master/hack/istio/install-bookinfo-demo.sh#L82-L85 and https://github.com/kiali/kiali/blob/master/hack/istio/install-bookinfo-demo.sh#L308-L309

Once you have a `--minikube-profile` option that optionally sets MINIKUBE_PROFILE, you can have this line be:
```suggestion
    export INGRESS_HOST=$(minikube -p ${MINIKUBE_PROFILE:-minikube} ip)
```